core/sys/darwin/Foundation/NSWindow.odin

```diff
@@ -716,3 +705,47 @@ Window_backingScaleFactor :: proc "c" (self: ^Window) -> Float {
 Window_setWantsLayer :: proc "c" (self: ^Window, ok: BOOL) {
 	msgSend(nil, self, "setWantsLayer:", ok)
 }
+@(objc_type=Window, objc_name="setIsMiniaturized")
+Window_setIsMiniaturized :: proc "c" (self: ^Window, ok: BOOL) {
+	msgSend(nil, self, "setIsMiniaturized:", ok)
+}
+@(objc_type=Window, objc_name="setIsVisible")
+Window_setIsVisible :: proc "c" (self: ^Window, ok: BOOL) {
+	msgSend(nil, self, "setIsVisible:", ok)
+}
+@(objc_type=Window, objc_name="setIsZoomed")
+Window_setIsZoomed :: proc "c" (self: ^Window, ok: BOOL) {
+	msgSend(nil, self, "setIsZoomed:", ok)
+}
+@(objc_type=Window, objc_name="isZoomable")
+Window_isZoomable :: proc "c" (self: ^Window) -> BOOL {
+    return msgSend(BOOL, self, "isZoomable")
+}
+@(objc_type=Window, objc_name="isResizable")
+Window_isResizable :: proc "c" (self: ^Window) -> BOOL {
+    return msgSend(BOOL, self, "isResizable")
+}
+@(objc_type=Window, objc_name="isModalPanel")
+Window_isModalPanel :: proc "c" (self: ^Window) -> BOOL {
+    return msgSend(BOOL, self, "isModalPanel")
+}
+@(objc_type=Window, objc_name="isMiniaturizable")
+Window_isMiniaturizable :: proc "c" (self: ^Window) -> BOOL {
+    return msgSend(BOOL, self, "isMiniaturizable")
+}
+@(objc_type=Window, objc_name="isFloatingPanel")
+Window_isFloatingPanel :: proc "c" (self: ^Window) -> BOOL {
+    return msgSend(BOOL, self, "isFloatingPanel")
+}
+@(objc_type=Window, objc_name="hasCloseBox")
+Window_hasCloseBox :: proc "c" (self: ^Window) -> BOOL {
+    return msgSend(BOOL, self, "hasCloseBox")
+}
+@(objc_type=Window, objc_name="hasTitleBar")
+Window_hasTitleBar :: proc "c" (self: ^Window) -> BOOL {
+    return msgSend(BOOL, self, "hasTitleBar")
+}
+@(objc_type=Window, objc_name="orderedIndex")
+Window_orderedIndex :: proc "c" (self: ^Window) -> Integer {
+    return msgSend(Integer, self, "orderedIndex")
```

Comment:
Spaces were used instead of a tab.

Reply:
I don't know what makes me think this, and it's just a hunch, but I'm getting the distinct feeling that spaces were used.

Reply:
Sorry for the trouble and thanks for pointing this out. I have `"editor.insertSpaces": false` in VS Code settings, not sure what happened here.

Reply:
It's quite alright. This led me to investigate `-vet-tabs`, and I identified some possible issues with it here: #4076.

Reply:
Oh, nice, I wasn't using that flag, just found more indentation issues thanks to it, although in my other repo.